components/common/map/LocationSearch.tsx

```
@@ -0,0 +1,185 @@
+            </div>
+          ))}
+        </div>
+      )}
+    </div>
+  );
+};
+
+// Debounce utility function
+function debounce<T extends (...args: any[]) => any>(
+  func: T,
+  delay: number
```

ESLint

[error] Insert `,`

(prettier/prettier)